cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/instantsearch-config.js

```diff
@@ -369,10 +378,13 @@ function enableInstantSearch(config) {
                                 });
                             });
                         }
+                        // Calculate displayPrice
+                        var { price, calloutMsg } = calculateDisplayPrice(item);
+                        item.displayPrice = price;
+                        item.calloutMsg = calloutMsg;
 
                         // Master-level indexing
                         if (item.variants) {
-                            let price;
                             item.variants.forEach(variant => {
                                 price = variant.price[algoliaData.currencyCode]
```

Comment:
This line should be removed too, or it will override the `price` calculated above